modules/doc/content/getting_started/installation/install_libtorch.md

```diff
@@ -85,6 +107,27 @@ by the `setup_libtorch.sh` script.
 
 !alert-end!
 
+For conda-based installations the user can link to the conda-based libtorch libraries
+using the approach above (using a typical installation path within conda):
+
+```bash
+./configure --with-libtorch=${CONDA_PREFIX}/lib/python[!package!minimum_python]/site-packages/torch
```

Comment:
Are we meaning to use minimum_python instead of default_python? Just making sure!

Reply:
Also, it looks like this didn't render correctly.

Reply:
Aaaa right, default is more common! Great catch!

Reply:
Turns out the doc system truncates 3.10 to 3.1 for the default python so I'll go with the minimum which is 3.7.

Reply:
I also can't add this to a code block so for now on I'll have 3.10 with a warning that it might be different for them.

Reply:
dang... yeah we need to fix that